### PR TITLE
{bio}[foss/2023b] GROMACS v2024.3 w/ PLUMED 2.9.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso
CORE_CNT=16

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3112.skitty.os - Linux RHEL 9.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.9.18
See https://gist.github.com/boegel/dd34617a6190de1d3fec3e618f77a1b8 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3
CORE_CNT=16

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
interactive1 - Linux Rocky Linux 8.9, x86_64, AMD EPYC-Milan Processor (zen3), Python 3.6.8
See https://gist.github.com/bedroge/d33b6a9d309ab78a2952c0f7c89db7dd for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 24.04.1 LTS (Noble Numbat), x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.12.3
See https://gist.github.com/bedroge/bf960004849560b0bb5869fa37d181bf for a full test report.

File: easybuild/easyconfigs/g/GROMACS/GROMACS-2024.3-foss-2023b-PLUMED-2.9.2.eb [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+dependencies = [
+    ('Python', '3.11.5'),
+    ('SciPy-bundle', '2023.11'),
+    ('networkx', '3.2.1'),
+    ('mpi4py', '3.1.5'),
+    ('PLUMED', _plumedver),
+]
+
+# PLUMED 2.9.0 is compatible with GROMACS 2023; 2023.3 seems to work fine too
```
Comment (bedroge) [inline easybuild/easyconfigs/g/GROMACS/GROMACS-2024.3-foss-2023b-PLUMED-2.9.2.eb:69]: This needs to be updated. In my log I found:
```
WARNING There is no support in PLUMED version 2.9.2 for GROMACS 2024.3: NOTE: shell only version, useful when plumed is cross compiled
PLUMED patching tool

Available MD engines:
  gromacs-2020.7
  gromacs-2021.7
  gromacs-2022.5
  gromacs-2023.5
  gromacs-2024.2
  namd-2.12
  namd-2.13
  namd-2.14
  qespresso-5.0.2
  qespresso-6.2
  qespresso-7.0
  qespresso-7.2
```
so I guess it should be:
```suggestion
# PLUMED 2.9.2 is compatible with GROMACS 2024.2; 2024.3 seems to work fine too
```
?

**Review by bedroge — APPROVED** [review]
Lgtm

**Comment by bedroge** [comment]
Going in, thanks @boegel!

**Pull request merged**